Projects/Features/SearchFeature/Sources/ViewControllers/SongSearchResultViewController.swift

```diff
@@ -117,12 +142,12 @@ final class SongSearchResultViewController: BaseReactorViewController<SongSearch
 
         headerView.snp.makeConstraints {
             $0.height.equalTo(30)
-            $0.top.equalToSuperview().offset(56)
+            $0.top.equalToSuperview().offset(72) // 56 + 16
             $0.leading.trailing.equalToSuperview().inset(20)
         }
 
         collectionView.snp.makeConstraints {
-            $0.top.equalTo(headerView.snp.bottom)
+            $0.top.equalTo(headerView.snp.bottom).offset(16)
```

Comment:
<img width="805" alt="스크린샷 2024-06-20 오전 11 28 02" src="https://github.com/wakmusic/wakmusic-iOS/assets/60254939/47a67539-e2db-4e97-be44-5629550e50c7">

```suggestion
            $0.top.equalTo(headerView.snp.bottom).offset(8)
```
이 부분이 맞는지 모르겠는데 피그마상으론 8로 보여요

Reply:
저게 위 아래 간격 맞출려고 샴퓨님이랑 간격 조절한거에요

Reply:
아하 확인했습니당